Remarkable-Donkeys/bangazonWebApp
# Kn add payment type

**The Name of this branch is**
kn-paymentType

* **Please check if the PR fulfills these requirements**


* **What kind of change does this PR introduce?** (Bug Fix, Feature, Document Update...)
feature and new migration
Please note, that even though the delete button is present, but does not have correct functionality yet.



* **What Ticket is associated with this PR**
[Add Payment Type](https://github.com/Remarkable-Donkeys/bangazonWebApp/issues/7)


* **What is the new behavior?**  (if this is a feature change)
When a user is authorized they can add a payment type to their account.

* **List all core functionality**
Adding a payment to the account.

* **Other information / Steps to Test**:
1. `git fetch --all`
2. `git checkout kn-paymentType`
3. A new migration was made in order to add an Active property to Payment Type, run `Update-Database` in Package Manager Console
4. Run application
5. Log-in as a user
6. go to {localhost}/PaymentType
7. Click Add Payment Option and add a new payment type for that user. Once the payment type has been add, you will be brought back to the main Payment Type page.
8. Log-out and repeat steps 5 & 6 as a different users. You should not be able to see the payment type you added for the first user.
9. If you log-out, and try to navigate to {localhost}/PaymentType (when you're NOT logged in) you should be brought to the log-in screen


***At least one person needs to approve pull request before merged to master. Indicate if you think more than one person should test the feature***
